test/test_filo.c

```diff
@@ -30,26 +33,27 @@ int main(int argc, char* argv[])
     write(fd, buf, strlen(buf));
     close(fd);
   } else {
+    rc = TEST_FAIL;
     printf("Error opening file %s: %d %s\n", filename, errno, strerror(errno));
   }
 
   char dest_filename[256];
   sprintf(dest_filename, "./testfile_%d.out", rank);
 
-  Filo_Init();
+  rc = Filo_Init();
```

Comment:
When checking the return codes of Filo calls, let's look to see whether we got FILO_SUCCESS and then set rc = TEST_FAIL if not.